src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -1328,7 +1328,7 @@ private Bid insertNetworkIdToMeta(Bid bid, int networkId, List<BidderError> erro
 
     private static boolean validatePrice(Bid bid) {
```

Comment:
Please consider to remove this method since we already have 0 price checking for deals in `org.prebid.server.validation.ResponseBidValidator#validateCommonFields`.